frontend/src/components/ProjectCardGrid/index.tsx

```diff
@@ -0,0 +1,33 @@
+import Project from '@models/Project'
+import ProjectCard, { IntersectionObserverRef } from '../ProjectCard'
+import styles from './index.module.scss'
+
+export const cardsPerRow = 3
+
+interface ProjectCardGridProps {
+  projects: Project[]
```

Comment:
```suggestion
  projects: Array<Project>
```

It's somewhat arbitrary but I've been trying to stick to one style. I'll add an eslint rule at some point.